LunaMary/src/Main.java

```diff
@@ -1,11 +1,42 @@
+import java.util.List;
 import java.util.Scanner;
 
 
 public class Main {
 
     public static void main(String[] args) {
 
-        Menu menu = new Menu();
+        OptionAsk optionAsk1 = new OptionAsk("Haces pasar a una chica que crees que baja después que tú…", 10, "A", false);
+        OptionAsk optionAsk2 = new OptionAsk("Te pones en un rincón", 10, "B", false);
+        OptionAsk optionAsk3 = new OptionAsk("Saludas a todo el mundo y te pones a hablar", 10, "C", false);
+        OptionAsk optionAsk4 = new OptionAsk("Te das cuenta de que una persona anda algo tocada", 10, "D", false);
+
+        OptionAsk optionAsk5 = new OptionAsk("Llamas a emergencias y procuras que la gente no se le agolpe", 10, "A", false);
+        OptionAsk optionAsk6 = new OptionAsk("Le preguntas lo que ha pasado", 10, "B", false);
+        OptionAsk optionAsk7 = new OptionAsk("Le das la mano hasta que llega ayuda", 10, "C", false);
+        OptionAsk optionAsk8 = new OptionAsk("Te interesas por cómo se siente y procuras arroparla", 10, "D", false);
+
+        List<OptionAsk> optionAskList1 = null;
```

Comment:
As specified on the constructor comment you always need to initialize your lists. Using a constructor of a class that is Instantiable like ArrayList because List is only an interface.